web/src/frontend/src/digisos/skjema/personopplysninger/index.tsx

```diff
@@ -71,6 +76,8 @@ class Personopplysninger extends React.Component<Props, OwnProps> {
 				</DigisosSkjemaSteg>
 			);
 		}
+
```

Comment:
Disse kommentarene bør fjernes. Enten sletter vi koden, eller så kan vi evt lage en oppgave på å slette den hvis det krever mer analyse for å se om det brukes :)

Reply:
Fjerner